src/utils/chain.js

```diff
@@ -188,7 +188,7 @@ export const supportedChains = {
     rpc_url: 'https://rpc-mainnet.maticvigil.com/',
     abi_api_url:
       'https://api.polygonscan.com/api?module=contract&action=getabi&address=',
-    tokenlist_api_url: 'https://api.polygonscan.com/api',
+    tokenlist_api_url: 'https://api.harmony.one',
```

Comment:
was updating the polygon api url a mistake?

Reply:
yes that was a fat finger, i'll change it back

Reply:
reverted back to the tokenlist api for polygon